CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   (docs/DesignDocs/system_ocall_opt_in.md#how-to-port-your-application) for more information.
```

Comment:
Nit: GitHub markdown doesn't see this as a link because there is a line break between the `](` ... the line break can be inserted after the opening parenthesis on the previous line.

Reply:
Thanks!